.github/ISSUE_TEMPLATE/proposing-new-components---.md

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
Is this file a duplicate of `.github/ISSUE_TEMPLATE/proposing-new-components.md`?